microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Verify your company account information
+description: Learn how to verify your account information when your company enrolls in the Microsoft Edge program on Partner Center to publish extensions to the Microsoft Edge Add-ons website.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/31/2022
+---
+# Verify the company account information associated with your Microsoft Partner Center account
+<!-- 
+or (would have to change 3-5 spots):
+# Verify your program information for Microsoft Edge Partner Center
+or [Added by Ashish]
+Verify the account information associated with the Microsoft Edge program of your Microsoft Partner Center account
+-->
+
+When you enroll in the Microsoft Edge program on Partner Center to publish extensions or change legal details in your profile, Microsoft verifies the information that you provide. Information can include your company name, company address, and primary contact details. During this process, Microsoft may send email to your primary contact to request verification.
+
+You can go to [Legal info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer) in Partner Center to monitor verification status for the Microsoft Edge Developer program.
+
+When account verification is complete, you can use your Partner Center dashboard to publish your extension to the Microsoft Edge Add-ons website.
+
+> [!NOTE]
+> Verification usually takes 3-5 business days. If more than five days have passed, you can contact support for assistance.
+
+
+<!-- ====================================================================== -->
+## What is verified and what is required?
+
+| Type of verification | What's verified | Suggestions |
+|---|---|---|
+| Email ownership | Email ownership verifies that the primary contact (primary email) address is valid. <br> * The primary contact email address must be a work account that is monitored and can send/receive email. <br> * Avoid using a personal email address not associated with the company domain, or a tenant user credential not associated with email. (For example, jsmith@testcompany.onmicrosoft.com or abc@gmail.com). | Be sure to flag email from Microsoft.com as a **safe** domain, and check junk email folders. <br> <br> If you don't receive the email ownership verification email message within one business day, you can ask us to send the email again. On to your Partner Center account, go to **Home > Account settings**. On the **Account settings l My profile** page, select **Resend verification email**. |
```

Comment:
The bar next to Account settings reads like an "I" 

Suggestion: Instead of Account settings | My profile please change to Account settings > My profile to increase readability.

Reply:
changed to \| to retain table format and display pipe symbol